product_attribute_set/tests/test_product_attribute.py

```diff
@@ -16,7 +16,7 @@ def test_write_attribute_values_text(self):
 
```

Comment:
I think this test can be remove.
we already test such thing in the base module

Reply:
We don't really test to **write** on the created attribute's fields (in the attribute_set's tests) as we do on this one... Should we ?
For the moment we only create attributes and check if they are in the form view with all the correct required and invisible attrs... Tell me if it is necessary

Reply:
Move this test in the base module